ember-cli/2017-07/july-27.md

```diff
@@ -0,0 +1,10 @@
+# ember-cli Meeting Notes - 2017-06-22
+
+## Attendees
+
+
+
+## Agenda
```

Comment:
on behalf of @cibernox: https://github.com/ember-fastboot/ember-cli-fastboot/pull/457